components/automate-ui/src/app/page-components/profile/profile.component.html

```diff
@@ -1,39 +1,46 @@
-<button class="profile profile-dropdown-toggle" (click)="toggleDropdown($event)" data-cy="user-profile-button">
-  <div class="user">
-    <div class="account-icon-container">
-      <chef-icon id="account-icon">person</chef-icon>
-    </div>
-    <div class="user-name profile-dropdown-toggle">{{ displayName }}</div>
-  </div>
+<button
+  aria-label="User Menu"
+  class="dropdown-toggle"
+  (click)="toggleDropdown($event)"
+  data-cy="user-profile-button">
+  <chef-icon aria-hidden="true" class="dropdown-toggle-user-icon">person</chef-icon>
+  <chef-icon aria-hidden="true">keyboard_arrow_down</chef-icon>
 </button>
 
 <chef-click-outside omit="profile-dropdown-toggle" (clickOutside)="closeDropdown()">
-  <ul class="logout-dropdown" *ngIf="dropdownVisible">
-    <app-authorized [allOf]="['/auth/users/{username}', 'get', userName]">
-      <li>
-        <a [routerLink]="['/user-details', userName]" (click)="closeDropdown()">Your Profile</a>
+  <chef-dropdown class="dropdown" [attr.visible]="dropdownVisible" (keydown.esc)="closeDropdown()">
+    <ul class="dropdown-list">
+      <li class="dropdown-list-item">
+        Signed in as <strong class="dropdown-display-name">{{ displayName }}</strong>
       </li>
-    </app-authorized>
-    <li>
-      <button (click)="showWelcomeModal()" data-cy="welcome-modal-button">About Chef Automate</button>
-    </li>
-    <li>
-      <button class="license-link" (click)="openLicenseModal()">License Information</button>
-    </li>
-    <li>
-      <a href="https://automate.chef.io/release-notes/?v={{ buildVersion }}" class="version-link" target="_blank">Chef Automate {{ buildVersion }}</a>
-    </li>
-    <li><chef-button primary (click)="logout()" data-cy="sign-out-button">Sign Out</chef-button></li>
-  </ul>
+      <app-authorized [allOf]="['/auth/users/{username}', 'get', userName]">
```

Comment:
the updates here will need to reflected in our cypress tests (which are unfortunately not run in the PR `verify` pipeline yet so has to be tested locally)

Running the login spec locally, looks like this bit needs to be updated:
https://github.com/chef/automate/blob/master/e2e/cypress/integration/01_login.spec.js#L85-L92

Reply:
We should really add a link to this in the automate-ui readme in the testing section.